doc/source/dev/contributor/pep8.rst

```diff
@@ -25,7 +25,7 @@ compliance before pushing your code:
 
    ::
 
-      flake8 scipy/optimize/linprog.py
```

Comment:
This file no longer exists, hence the update.

Reply:
Actually, we might want to remove the mention of direct `flake8` use, unless we also want to explain what options need to be passed.  By default, it no longer report issues because of the additions to `tox.ini`.

Reply:
It would be fine if it's explained in more detail. But currently, this states that it is OK to use `flake8` with default settings. That is no longer accurate after gh-14456, so I think something needs to change. I suggested removing it because that would be the simplest option, but yes, another possible way is to explain in more detail. Your choice.

Reply:
Done (removed flake8 example but it still mentions that we use flake8 under the hood).